src/wallet/wallet.h

```diff
@@ -1193,6 +1193,17 @@ class WalletBatchScanner : public BatchScanner {
         const int nHeight);
 };
 
+/**
+ * We have interfaces that only accept an address, so we sometimes use a UA as a proxy for the
+ * associated account. This indicates whether the proxy should allow all receivers or only
+ * non-transparent ones.
+ */
+enum AccountProxy {
+    NonTransparent, //<! Can only send from non-transparent receivers in the account.
+    SingleTransparent, //<! Can send from a single transparent address in the account.
+    Full, //<! Can send from any combination of receivers in the account.
```

Comment:
If this were renamed to `UnifiedAccountSpendingPolicy` then maybe `Full` could be renamed/elaborated; maybe `NonTransparentOnly`, `SingleTransparentOnly`, `Any` ?

Reply:
I gave them different names than you suggested. Let me know what you think.